mmv1/third_party/terraform/services/dataplex/resource_dataplex_glossary_test.go.tmpl

```diff
@@ -0,0 +1,76 @@
+package dataplex_test
+
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+)
+
+func TestAccDataplexGlossary_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataplexGlossary_update(context),
+			},
+			{
+				ResourceName:            "google_dataplex_glossary.glossary_test_id",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"glossary_id", "labels", "location", "terraform_labels"},
+			},
+			{
+				Config: testAccDataplexGlossary_full(context),
+				ConfigPlanChecks: resource.ConfigPlanChecks{
+					PreApply: []plancheck.PlanCheck{
+						plancheck.ExpectResourceAction("google_dataplex_glossary.glossary_test_id", plancheck.ResourceActionUpdate),
+					},
+				},
+			},
+			{
+				ResourceName:            "google_dataplex_glossary.glossary_test_id_full",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"glossary_id", "labels", "location", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccDataplexGlossary_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+	resource "google_dataplex_glossary" "glossary_test_id_full" {
+  		glossary_id = "tf-test-glossary-full%{random_suffix}"
+  		location     = "us-central1"
+		labels = { "tag": "test-tf" }
+		display_name = "terraform glossary"
+		description = "glossary updated by Terraform"
```

Comment:
these should have different values (or no values) to make sure that they will get updated in step 3.
```suggestion
```